tests/generation/test_utils.py

```diff
@@ -3252,6 +3252,28 @@ def test_default_max_length_warning(self):
             model.generate(input_ids)
             self.assertEqual(len(warning_list), 0)
 
+    def test_length_warning_assisted_generation(self):
```

Comment:
There should also be a test that `min_new_tokens` parameter behaves as expected, especially when `max_new_tokens` is also set and when it's not set at all i.e. goes to default value

Reply:
Added one more test checking if the length is in range between min and max lengths